project_setup.md

```diff
@@ -57,7 +56,8 @@ help you decide which tool to use for packaging.
 - The documentation is set up with the ReadTheDocs Sphinx theme.
   - Check out its [configuration options](https://sphinx-rtd-theme.readthedocs.io/en/latest/).
 - [AutoAPI](https://sphinx-autoapi.readthedocs.io/) is used to generate documentation for the package Python objects.
```

Comment:
do we want to use sphinx or mkdocs?

Reply:
Sphinx is used by the template. I don't really know the differences, so I don't have a preference here...